tests/baselines/reference/checkObjectDefineProperty.errors.txt

```diff
@@ -0,0 +1,73 @@
+tests/cases/conformance/jsdoc/validate.ts(13,3): error TS2540: Cannot assign to 'lastName' because it is a constant or a read-only property.
+tests/cases/conformance/jsdoc/validate.ts(14,3): error TS2540: Cannot assign to 'houseNumber' because it is a constant or a read-only property.
+tests/cases/conformance/jsdoc/validate.ts(15,1): error TS2322: Type '12' is not assignable to type 'string'.
+
+
+==== tests/cases/conformance/jsdoc/validate.ts (3 errors) ====
+    // Validate in TS as simple validations would usually be interpreted as more special assignments
+    import x = require("./");
+    x.name;
+    x.lastName;
+    x.zip;
+    x.houseNumber;
+    x.zipStr;
+    
+    x.name = "Another"; 
+    x.zip = 98123;
+    x.zipStr = "OK";
+    
+    x.lastName = "should fail";
+      ~~~~~~~~
+!!! error TS2540: Cannot assign to 'lastName' because it is a constant or a read-only property.
+    x.houseNumber = 12; // should also fail
+      ~~~~~~~~~~~
+!!! error TS2540: Cannot assign to 'houseNumber' because it is a constant or a read-only property.
+    x.zipStr = 12; // should fail
+    ~~~~~~~~
+!!! error TS2322: Type '12' is not assignable to type 'string'.
+    
+==== tests/cases/conformance/jsdoc/index.js (0 errors) ====
+    const x = {};
+    Object.defineProperty(x, "name", { value: "Charles", writable: true });

```

Comment:
you should have a value test that leaves off writable, if that's allowed. Actually, you should have a test with value left out too. And maybe a test with some of the properties we don't care about?

Also a test with non-literals, just in case that hurts our ... error reporting or something?

According to [MDN's Object.defineProperty page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty), we should have an error if both get/set and value are present. I didn't see a test for that either.

Also, who spells 'writeable' as 'writable'?
Later:everybody else, it turns out.